persistence/src/main/scala/com/lightbend/lagom/internal/persistence/cassandra/CassandraConfig.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence.cassandra
+
+import scala.language.implicitConversions
+
+import akka.actor.ActorSystem
+import javax.inject.Inject
+import javax.inject.Provider
+import javax.inject.Singleton
+
+trait CassandraConfig {
+  def uris: Set[(String, String)]
+}
+
+object CassandraConfig {
```

Comment:
Mmm, forget what I said, it's all good since the tcp address hardcoding is happening iff `session-provider` is `ServiceLocatorSessionProvider`